src/scripts/EasternKingdoms/zone_ironforge.cpp

```diff
@@ -395,17 +435,17 @@ class npc_steamcrank : public CreatureScript
         }
 };
 
-#define MEK_1_0    "Пусть они заберут наши жизни, но им никогда не забрать..."
-#define MEK_1_1    "...нашу ИЗОБРЕТАТЕЛЬНОСТЬ!"
-#define LIS_1_0    "Что? Я понятия не имею о чем он говорит! Это ужастно!"
-#define MEK_2_0    "Мы не позволим себя уничтожить! Мы не сдадимся без боя!"
-#define MEK_2_1    "Мы будем жить! Мы будем продолжать жить! Сегодня мы празднуем..."
-#define MEK_2_2    "...наш День Независимости!"
-#define LIS_2_0    "Кошмар! Хотя... может и ничего, если немного подчистить."
-#define MEK_3_0    "Вы должны тщательно обыскать каждую заправку, все дома, склады, фермы, уборные и конуры в этом районе."
-#define MEK_3_1    "Имя беглеца - Анжинер Термоштепсель."
-#define MEK_3_2    "Идите и задержите его."
-#define LIS_3_0    "Пожалуй, должно работать. Хотя чего-то ему явно не хватает."
+#define MEK_1_0 "Let them take away our lives, but they will never take ..."
+#define MEK_1_1 "... our ingenuity!"
+#define LIS_1_0 "What? I have no idea what he was saying! It uzhastno!"
+#define MEK_2_0 "We will not let themselves be destroyed! We will not surrender without a fight!"
+#define MEK_2_1 "We will live! We will continue to live! Today we celebrate ..."
+#define MEK_2_2 "... our Independence Day!"
+#define LIS_2_0 "Nightmare! Although ... maybe nothing, if a little clean up."
+#define MEK_3_0 "You must carefully search every gas station, all the houses, warehouses, farms, kennels and toilets in the area."
+#define MEK_3_1 "fugitive Name - Mekgineer Thermaplugg."
+#define MEK_3_2 "Go and hold it."
+#define LIS_3_0 "Perhaps it should work. Though something it is clearly not enough."
```

Comment:
As i said it's not my work i only adapted so it can compile and "work" but it needs re-work and mass testing